cyber-resilience-sig/minutes/faq-task-force/2025-04-04-mom-faq-tf.md

```diff
@@ -10,3 +10,88 @@
 | Split-up the work | Tobie | 20 |
 | Decide on meeting cadence | Tobie | 5 |
 | AOB | | 5 |
+
+
+## Apr 4, 2025 | [FAQ Task Force](https://www.google.com/calendar/event?eid=NDJybzZmaW9zamlpZ2xoaXVvMDJrYzgxZjlfMjAyNTA0MDRUMTEzMDAwWiB0b2JpZS5sYW5nZWxAZWNsaXBzZS1mb3VuZGF0aW9uLm9yZw)
+
+## Attendees: 
+
+* Tobie Langel \[tobie\] (Eclipse/UnlockOpen)  
+* Roman Zhukov (Red Hat)  
+* Dick Brooks (Business Cyber Guardian)  
+* Salve J. Nilsen \[sjn\] (CPANSec)  
+* Paolo Vecchi (The Document Foundation/Luxembourg OSPO)  
+* Jakub Zelenka \[bukka\] (The PHP Foundation)  
+* Olle E. Johansson \[oej\] (Edvina/OWASP)  
+* Maarten Aertsen \[maertsen\] (NLnet Labs foundation)  
+* Juan Rico (Eclipse Foundation)  
+* Götz Görisch \[GoetzGoerisch\] \- (Verein Deutscher Werkzeugmaschinenfabriken e.V. / German Machine Tools builders association)  
+* Alistair Woodman (Erlang Ecosystem Foundation (EEF))  
+* Dirk-Willem van Gulik \[dirkx\] (Apache Software Foundation)  
+* Timo Perälä (Nokia)  
+* Jordan Maris \[jmaris\] (OSI)  
+* Victor Roland (Obeo)  
+* Sal Kimmich
+
+## Agenda:
+
+[https://github.com/orcwg/orcwg/blob/main/cyber-resilience-sig/minutes/faq-task-force/2025-04-04-mom-faq-tf.md](https://github.com/orcwg/orcwg/blob/main/cyber-resilience-sig/minutes/faq-task-force/2025-04-04-mom-faq-tf.md)
+
+## Notes:
+
+- FAQ timeline:  
+  - Target date for a v1.0 release of the FAQ \- end of May  
+  - Document should list contributors and members of the ORCWG  
+- Maturity level  
+  - Processes to be documented:  
+    - How to use GH  
+    - SIG review process (async)  
+    - Publication process  
+- FAQ Project Board  
+  - Updating category names is possible, to do it we will use issues to discuss the proposal.  
+- Split-up the work  
```

Comment:
Perhaps some of this should be reflected in the notes, or in the notes of today's meeting:

https://orcwg.slack.com/archives/C08842CL6N7/p1744011917673339